src/phoenix/trace/spans_dataframe_utils.py

```diff
@@ -0,0 +1,83 @@
+from enum import Enum
+
+import pandas as pd
+
+
+class SpansDataframeFormats(Enum):
+    """
+    An enumeration of the different formats the spans dataframe can be in
+    Notably, it can be beneficial to convert the dataframe to other formats
+    to pipe the values into other functions like LLM Evals
+    """
+
+    key_value = "key_value"
+    """
+    A dataframe of root spans that captures the input and output of each trace
+
+    Example:
+    +---------+----------------+----------------+
+    | span_id | input          | output         |
+    +---------+----------------+----------------+
+    | span_1  | why?           | because        |
+    +---------+----------------+----------------+
+    """
+
+
+def get_root_spans(spans_df: pd.DataFrame) -> pd.DataFrame:
+    """
+    Returns a dataframe of root spans
+
+    Parameters
+    __________
+    spans_df: pd.DataFrame
+        the dataframe of spans
+    """
+    df = spans_df.copy()
+    return df[df["parent_id"].isna()]
+
+
+def to_format(spans_df: pd.DataFrame, format: SpansDataframeFormats) -> pd.DataFrame:
```

Comment:
Are `SpansDataframeFormats` intended to also handle exploding of documents for retriever spans, etc.?

Reply:
Potentially yeah. We could normalize to something similar to what ragas consumes.